packages/flutter/lib/src/services/autofill.dart

```diff
@@ -649,9 +649,9 @@ class AutofillConfiguration {
   /// {@template flutter.services.autofill.autofillHints}
   /// For the best results, hint strings need to be understood by the platform's
   /// autofill service. The common values of hint strings can be found in
-  /// [AutofillHints], as well as the platforms that understand each of them.
+  /// [AutofillHints], as well as their availability on different platforms.
```

Comment:
looking for a better word for "availability"

Reply:
I think availability is pretty good choice here. Alternatively you could say "how well they're supported on different platforms".

Reply:
111

Reply:
111

Reply:
11111